raster/r.compress/main.c

```diff
@@ -96,7 +96,7 @@ int main(int argc, char *argv[])
 
     stat = EXIT_FAILURE;
```

Comment:
The logic needs to be reversed differently: `process()` returns 0 on success, 1 on failure. We expect success. If any call to `process()` fails, we need to exit with `EXIT_FAILURE`. Thus
```
    stat = EXIT_SUCCESS;
    for (n = 0; (name = map->answers[n]); n++)
	if (process(name, uncompress->answer))
	    stat = EXIT_FAILURE;
```

Reply:
@metzm Thanks for the explanation. 
I am changing the code accordingly.